agent/agent.go

```diff
@@ -447,6 +447,17 @@ func stopServiceInputs(inputs []*models.RunningInput) {
 	}
 }
 
+// stopRunningOutputs stops all running outputs.
+func stopRunningOutputs(outputs []*models.RunningOutput) {
+	for _, output := range outputs {
+		log.Printf("I! [agent] stop running output: %s", output.LogName())
```

Comment:
```suggestion
```

Reply:
I have removed this log message and I think is better to call "output.Close()" instead of "output.Output.Close()"